ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthServer.java

```diff
@@ -47,23 +47,53 @@
 class EthServer {
   private static final Logger LOG = LogManager.getLogger();
 
+  static final int DEFAULT_REQUEST_LIMIT = 200;
```

Comment:
I'd cut this since its only getting used in one test util - just set the values manually there.

Reply:
fixed